frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1574,6 +1595,32 @@ With this feature enabled, the current page is factored in, resulting in the cou
             {
                 text = _("Item max width"),
                 sub_item_table = {
+                    {
+                        text_func = function()
+                            return T(_("Book-authors item: %1".. "\u{202F}%"), self.settings.book_authors_max_width_pct)
+                        end,
+                        callback = function(touchmenu_instance)
+                            local items = SpinWidget:new{
+                                value = self.settings.book_authors_max_width_pct,
+                                value_min = 10,
+                                value_step = 5,
+                                value_hold_step = 20,
+                                value_max = 100,
+                                unit = "%",
+                                title_text = _("Book-authors item"),
+                                info_text = _("Maximum percentage of screen width used for book-authors"),
+                                book_authors_max_width_pct = 30,
```

Comment:
This line is wrong.
I see that you have copied it from the other item but it is wrong there too.
There must be the default value passed to the SpinWidget instead.

Reply:
So what's the fix? Is there an example?